arrows/ocv/image_container.cxx

```diff
@@ -304,8 +324,9 @@ image_container_to_ocv_matrix(const vital::image_container& img, image_container
   if( const ocv::image_container* c =
           dynamic_cast<const ocv::image_container*>(&img) )
   {
-    if(cm == image_container::RGB)
+    if(cm == image_container::RGB || cm == image_container::OTHER_COLOR_MODE || result.channels()<3 || result.channels() > 4)
```

Comment:
It might be better to use `cm != image_container::BGR` instead of `cm == image_container::RGB || cm == image_container::OTHER_COLOR_MODE` both because it is more compact and also because the latter will break if more color modes are added in the future.